openmc/mgxs/library.py

```diff
@@ -819,20 +820,22 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             name += '_' + nuclide
         name += '.' + xs_id
         xsdata = openmc.XSdata(name, self.energy_groups)
-        if order is 0:
-            xsdata.order = order
+
+        if order is None:
+            # Set the order to the Library's order (the defualt behavior)
+            xsdata.order = self.legendre_order
         else:
-            msg = 'Generating anisotropic scattering from openmc.Library' \
-                  'objects has not yet been implemented.'
-            raise NotImplementedError(msg)
+            # Set the order of the xsdata object to the minimum of
+            # the provided order or the Library's order.
+            xsdata.order = min(order, self.legendre_order)
```

Comment:
I'm not sure I see the contradiction.  I edited the docstring slightly but it still says the same thing: ```order : Scattering order for this data entry.  Default is None,
            which will set the XSdata object to use the order of the
            Library.```
This is done by the code in the block before the set of line comments we are discussing now:
```
if order is None:
    # Set the order to the Library's order (the defualt behavior)
    xsdata.order = self.legendre_order```